Update init

cwb124 opened the pull request:
Safe syntax was wrong and causing errors when deploying postgresql services.  Removed the '-l' safe flag that signified a length for the username.  The flag isn't needed, just the number desired.  Would get this error in the blacksmith log when deploying postgresql:
`2018-08-29 18:48:51.139 DEBUG  init script `/var/vcap/data/blacksmith/postgresql-forge/services/standalone/init' said:
USAGE: gen [length] path key`

Also replaced redis with postgresql in the comments to avoid confusion.

cwb124 committed:
Update init

Safe syntax was wrong and causing errors when deploying postgresql services.  Removed the '-l' safe flag that signified a length for the username.  The flag isn't needed, just the number desired.  Would get this error in the blacksmith log when deploying postgresql:
`2018-08-29 18:48:51.139 DEBUG  init script `/var/vcap/data/blacksmith/postgresql-forge/services/standalone/init' said:
USAGE: gen [length] path key`

Also replaced redis with postgresql in the comments to avoid confusion.

jhunt commented:
I believe this came down to an issue with the `safe` from toolbelt interfering with the `safe` from blacksmith.

jhunt commented:
@cwb124 have you tried updating toolbelt?

cwb124 commented:
What toolbelt?  And how might I update it?

jhunt commented:
The addon, probably via the runtime configs.

cwb124 commented:
I updated the toolbelt to 3.4.2, applied rc to bosh director.  Redeployed blacksmith and updated service-broker.  Still same error.

2018-11-27 20:59:37.293 DEBUG init script `/var/vcap/data/blacksmith/postgresql-forge/services/standalone/init' said:
USAGE: gen [length] path key

cwb124 commented:
I removed the -l as a workaround but it's very fragile and the Init file edit doesn't survive a blacksmith service restart.

jhunt commented:
Can you exempt the blacksmith VM from the toolbelt runtime config?  I still believe this is an unfortunate collision between toolbelt and blacksmith, specifically with the old version of safe that is (still!) in toolbelt latest.

jhunt commented:
@cwb124 did removing toolbelt help?
